dir/path.go

```diff
@@ -79,21 +79,35 @@ const (
 
 var userConfigDir = os.UserConfigDir // for unit test
 
-func init() {
-	loadUserPath()
-}
-
 // loadUserPath function defines UserConfigDir and UserLibexecDir.
 func loadUserPath() {
-	// set user config
-	userDir, err := userConfigDir()
-	if err != nil {
-		panic(err)
+	UserConfigDir = ""
+	UserLibexecDir = ""
```

Comment:
The string default value is empty string, so we may don't need this set the empty value?

Reply:
This was more to reset the values but I've removed the func in [46c35d4](https://github.com/notaryproject/notation-go/pull/449/commits/46c35d45a2c830ad16861f7ab00b6a9ccb8aec9d).